calculator_app/apps.py

```diff
@@ -4,3 +4,6 @@
 class CalculatorAppConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'calculator_app'
+
+    def ready(self):
+        import calculator_app.signals
```

Comment:
Use signals as a **last** resort.

Long answer:

Often when new Djangonauts discover signals, they get signal-happy. They start sprinkling signals everywhere they can and feeling like real experts at Django.

After coding this way for a while, projects start to turn into confusing, knotted hairballs that can’t be untangled. Signals are being dispatched everywhere and hopefully getting received somewhere, but at that point it’s hard to tell what exactly is going on.